cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/DatabasesTest.scala

```diff
@@ -140,12 +152,38 @@ class DatabasesTest extends DocumentingTest with QueryStatisticsTestSupport {
           statsOnlyResultTable()
         }
         p("For more details on primary and secondary server roles, see <<operations-manual#clustering-introduction-operational, Cluster overview>>.")
+        note {
+          p("The `TOPOLOGY` are only available to standard databases and not composite databases.")
+        }
+      }
+      section("Creating composite databases", "administration-databases-create-composite-database", "enterprise-edition") {
+        // TODO: add a link to the place explaining what composite databases are
+        p(
+          """Composite databases are a type of database that instead of containing data holds other databases though aliases.""".stripMargin)
```

Comment:
```suggestion
          """Composite databases are a type of database that instead of containing data holds references to other databases that can be queried together through aliases.""".stripMargin)
```
Or words to this effect, I'm not sure I like my version too much either.